dwave/system/samplers/leap_hybrid_sampler.py

```diff
@@ -971,12 +977,20 @@ def hook(model, future):
         model.states.from_future(future, hook)
 
         def collect():
-            timing = future.timing
-            for msg in timing.get('warnings', []):
+            timing = future.timing.copy()
+            info = dict(
+                timing=timing,
+                warnings=timing.pop('warnings', []),
+                # match SampleSet.info fields (see :meth:`~dwave.cloud.computation.Future._get_problem_info`)
+                problem_id=future.id,
```

Comment:
would be so good to have "job" or "submission" versus "problem" in such places

Reply:
I agree, and I opened a feature request to rename problem to job, bqm id to problem (or data), etc.. but it's a very pervasive change, so using `problem_data_id` and `problem_id` is the best we can do right now for a sort of consistency and a partial sanity.